packages/ui-components/src/components/Map/Markers/CircleProportionMarker.js

```diff
@@ -16,39 +16,35 @@ const HoverCircle = styled(CircleMarker)`
   }
 `;
 
-export const CircleProportionMarker = React.memo(
```

Comment:
Remove the radius scale handling here as it is handled when the data is initially parsed now